src/components/ContactForm/ContactForm.jsx

```diff
@@ -1,36 +1,30 @@
-import './ContactForm.scss';
+// import './ContactForm.scss';
 import React from 'react';
 
 export default function ContactForm() {
   return (
-    <form name="contact" method="POST" data-netlify="true">
-      <div className="formInputs">
+    <form className="flex flex-col md:flex-row md:space-x-20" name="contact" method="POST" data-netlify="true">
+      <div className="flex flex-col md:flex-1">
         <input type="hidden" name="form-name" value="contact" />
-
-        <input
-          type="text"
-          name="name"
-          placeholder="Votre nom"
-          required
-        />
-        <input
-          type="email"
-          name="email"
-          placeholder="Votre adresse mail"
-          required
-        />
-        <textarea
-          name="message"
-          placeholder="Votre message"
-          required
-        />
+        <div className="relative mt-14">
+          <input className="peer w-full bg-white border-x-0 border-t-0 border-b-2 border-black px-0 pt-0 pb-1 h-10 text-base focus:ring-0 focus:border-black placeholder-transparent" type="text" name="name" id="name" placeholder="Your name" required />
+          <label htmlFor="name" className="absolute left-0 -top-5 text-gray-600 text-sm cursor-text transition-all peer-placeholder-shown:text-base peer-placeholder-shown:text-gray-500 peer-placeholder-shown:top-1 peer-focus:-top-5 peer-focus:text-gray-600 peer-focus:text-sm">Your name</label>
+        </div>
+        <div className="relative mt-16">
+          <input className="peer w-full bg-white border-x-0 border-t-0 border-b-2 border-black px-0 pt-0 pb-1 h-10 text-base focus:ring-0 focus:border-black placeholder-transparent" type="email" name="email" id="email" placeholder="Your email address" required />
```

Comment:
@jc3lee  Do you think we could use @apply to create a specific groups of similar inputs to make it more "readable" : https://tailwindcss.com/docs/reusing-styles#extracting-classes-with-apply ? Is it good idea ?

Reply:
Haha, I knew you'd notice 😁. Yes we could indeed make it less repetitive and more readable with `apply`. The other way is to use components instead. Tailwind recommends not to use apply as much with react since we could achieve the same with components but both ways are fine. I should have explained earlier but I intend to refactor using components in the next issue as I update the code to use ts 👌.